Add CS faculty members in Shenzhen University

jihuomail opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x]  Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x]  Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x]  If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x]  Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x]  If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x]  If the institution you are adding is not in the US,
update `country-info.csv`.

jihuomail committed:
Update country-info.csv

jihuomail committed:
Add files via upload

jihuomail committed:
Add files via upload

emeryberger commented on csrankings.csv:
```diff
@@ -2691,6 +2693,7 @@ Damian Niwinski,University of Warsaw,http://www.mimuw.edu.pl/~niwinski,JZ4771IAA
 Damien Muselet,Université Jean Monnet,http://perso.univ-st-etienne.fr/muda8804,TyNSxjQAAAAJ
 Damien Stehlé,Ecole Normale Superieure de Lyon,http://perso.ens-lyon.fr/damien.stehle,1qab5koAAAAJ
 Damien Vergnaud,Ecole Normale Superieure,http://www.di.ens.fr/~vergnaud,dt96T_wAAAAJ
+Daming Shi 0001,Shenzhen University,https://dblp.uni-trier.de/pers/s/Shi_0001:Daming,8nZ0ukUAAAAJ
```
DBLP is not a home page.

emeryberger commented on csrankings.csv:
```diff
@@ -3344,6 +3348,7 @@ Ding-Zhu Du,University of Texas at Dallas,http://www.utdallas.edu/~dxd056000,oAa
 Dingfu Zhou,Australian National University,https://researchers.anu.edu.au/researchers/zhou-d,pfBeKioAAAAJ
 Dinggang Shen,University of North Carolina,http://cs.unc.edu/people/dinggang-shen,v6VYQC8AAAAJ
 Dinghao Wu,Pennsylvania State University,https://faculty.ist.psu.edu/wu,Vxaxv10AAAAJ
+Dingming Wu,Shenzhen University,https://dblp.uni-trier.de/search?q=Dingming+Wu,oRbQlsEAAAAJ
```
DBLP is not a home page.

emeryberger commented on csrankings.csv:
```diff
@@ -4961,12 +4967,14 @@ Hao Che,University of Texas at Arlington,http://crystal.uta.edu/~hche,Vyrg19YAAA
 Hao Chen 0003,University of California - Davis,http://web.cs.ucdavis.edu/~hchen,1Aa3qxIAAAAJ
 Hao Huang,Nanjing University,http://www.mathcs.emory.edu/~hhuan30,bTTwjj8AAAAJ
 Hao Li 0015,University of Southern California,http://hao-li.com,NFeigSoAAAAJ
+Hao Liao,Shenzhen University,https://dblp.org/pers/hd/l/Liao:Hao,Tu5ZuREAAAAJ
```
As above, DBLP is not a valid home page.

emeryberger commented:
There are numerous DBLP entries in lieu of home pages. These all need to point to actual faculty home pages. There is also a conflict in `country-info.csv` that needs to be resolved.

emeryberger commented:
bump @jihuomail

jihuomail committed:
Add files via upload

remove faculties who do not have their own homepage

jihuomail commented:
@emeryberger Thanks for your comments.

I have removed those faculties from the list who do not have their own personal homepage, and replaced some dblp pages as mentioned. Could you please help check again?

emeryberger commented:
You apparently edited the file with Excel. That does not work; it corrupts many Google Scholar entries. Please edit with something else.

jihuomail committed:
Add files via upload

emeryberger commented:
Thanks; please resolve conflicts and I will merge.

jihuomail commented:
Hi @emeryberger I can not see any conflicts currently, how can I resolve the conflicts?

emeryberger commented:
Do a pull and you will see them.

jihuomail committed:
Add files via upload

jihuomail committed:
Add files via upload

jihuomail commented:
Hi @emeryberger Could you please help me resolve the conflicts (merge my branch with commit f9abefa)? I tried to follow your suggestions and found the conflicts in my local repository, after updating, it seemed that I made it worse...

emeryberger commented:
How about you close this PR and open a new one, forking the most recent repo?

jihuomail commented:
Thanks! I will do it.
